jenkins-integration/razor/README.md

```diff
@@ -0,0 +1,160 @@
+# Puppet Server Perf Testing - Razor Configuration
+
+This directory contains some files and a script that can be used to set up
+Razor policies for managing the blades that are used for Puppet Server perf
+testing.
+
+We have 4 blades allocated for our perf testing.  At the time of this writing,
+the hostnames are:
+
+* puppetserver-perf-sut54.delivery.puppetlabs.net
+* puppetserver-perf-sut56.delivery.puppetlabs.net
+* puppetserver-perf-sut57.delivery.puppetlabs.net
+* puppetserver-perf-driver55.delivery.puppetlabs.net
+
+More details on the hardware can be found [here](https://confluence.puppetlabs.com/display/PERF/Hardware+Assets).
+
+The three SUT boxes are intended to be where we install and run PE.  These nodes
+will need to have their operating systems wiped and re-installed frequently to
+ensure that we get clean data from run to run.  The "driver" box is where we
+will run Jenkins and Gatling to drive the perf tests.
+
+To make sure that this environment is reproducible and to make it easy to get
+the operating systems re-installed, we are using the SysOps team's razor instance,
+along with their Puppet modules, to manage the nodes.
+
+Docs on the Ops razor instance can be found [here](https://confluence.puppetlabs.com/display/OPS/Razor).
+
+There are four concepts in Razor that are relevant here:
+
+* brokers
+* tasks
+* tags
+* policies
+
+The brokers and tasks are managed via the Ops puppet modules (see
+`site/profile/manifests/razor.pp`).  The files in this directory can be used to
```

Comment:
At the time of this writing, that stuff hadn't been merged into their production branch.  I think maybe it has been now, so if you think it's worthwhile then I can check and link.